CHANGELOG.md

```diff
@@ -2,6 +2,79 @@
 
 All notable changes to this project are documented in this file.
 
+## 1.3.0
+
+**Release date:** 2024-05-03
+
+This minor release promotes the Helm APIs to GA, and comes with new features,
+improvements and bug fixes.
+
+### HelmRepository
+
+The `HelmRepository` API has been promoted from `v1beta2` to `v1` (GA).
+The `v1` API is backwards compatible with `v1beta2`.
+
+For `HelmRepository` of type `oci`, the `.spec.insecure` field allows connecting
+over HTTP to an insecure non-TLS container registry. 
+
+To upgrade from `v1beta2`, after deploying the new CRD and controller,
+set  `apiVersion: source.toolkit.fluxcd.io/v1` in the YAML files that
+contain `HelmRepository` definitions.
+Bumping the API version in manifests can be done gradually.
+It is advised to not delay this procedure as the beta versions will be removed after 6 months.
+
+### HelmChart
+
+The `HelmChart` API have been promoted from `v1beta2` to `v1` (GA).
+The `v1` API is backwards compatible with `v1beta2`, with the exception
```

Comment:
So it is not backward compatible right? I think a sentence about this with warning would be nice. Something like:

⚠️ **Warning:** The deprecated field `.spec.valuesFile` has been removed and will be silently dropped after upgrading to v1.3.0. Please make sure to upgrade to `.spec.ValuesFiles` only before upgrading.

Reply:
People can't do anything about this field as the HelmChart is generated by the HelmRelease. How to migrate would be documented in the helm-controller change log. Also this phasing is not new, this is how we've been documented deprecated fields removals in GA, see: https://github.com/fluxcd/flux2/releases/tag/v2.0.0.

Reply:
This is not blocking at all, if you think this is sufficient.

> People can't do anything about this field as the HelmChart is generated by the HelmRelease.

This will no longer be true, because they will be able to use `chartRef` after the next release. So they might come look for answers on this changeLog directly.

Reply:
But `chartRef` doesn't exist in Flux 2.2, it will only support HelmChart v1 which doesn't contain that field. No one would have HelmCharts v1beta2 created hand.